lib/core-integration/src/Test/Integration/Scenario/API/ByronWallets.hs

```diff
@@ -399,6 +407,21 @@ spec = do
             [ expectResponseCode @IO HTTP.status201
             , expectField (#balance . #available) (`shouldBe` Quantity faucetAmt)
             ]
+
+    it "BYRON_RESTORE_10 - Can restore wallet using master root key" $ \ctx -> do
+        m <- genMnemonics @12
+        let passw = "Secure Passphrase"
+        w <- emptyByronWalletWith ctx "random"
+             ("Byron Wallet", m, passw)
+        rd <- request
+              @ApiByronWallet ctx (Link.deleteWallet @'Byron w) Default Empty
+
+        let (Right seed) = fromMnemonic @'[12] m
+        let _rootXPrv =
+                generateKeyFromSeed seed (Passphrase $ BA.convert $ T.encodeUtf8 passw)
```

Comment:
@paweljakubas may you clarify what this call is for :thinking: ? Or is there actually an API call missing from here?

Reply:
yeah, this is not finished integration tests. I was waiting for your branch to me merged to use stuff hashing password

Reply:
Ah! Damn. I actually removed that from 'encryptPassphrase` because there's actually no reason we would try to encrypt a passphrase with Scrypt on the new codebase :s 

May I suggest to use this passphrase hash instead:

```
31347c387c317c574342652b796362417576356c2b4258676a344a314c
6343675375414c2f5653393661364e576a2b7550766655513d3d7c2f37
6738486c59723174734e394f6e4e753253302b6a65515a6b5437316b45
414941366a515867386539493d"

It was generated using cardano-sl, it corresponds to the passphrase "patate".